library/psa_crypto_core.h

```diff
@@ -64,9 +64,19 @@ typedef struct
     } data;
 } psa_key_slot_t;
 
-/** Flag for psa_key_slot_t::attr::core::flags indicating that the
- * slot is in use. */
-#define PSA_KEY_SLOT_FLAG_ALLOCATED ( (uint16_t) 0x0001 )
+/** Test whether a key slot is occupied.
+ *
+ * A key slot is occupied iff the key type is nonzero. This works because
```

Comment:
`iff` -> `if`

Reply:
No. Iff = “if and only if”

Reply:
Oh, right - Chris has reminded me of this acronym - if and only if. I really don't have it memorized.